source/content/global-cdn.md

```diff
@@ -32,6 +32,24 @@ We recommend installing the Pantheon Advanced Page Cache [plugin](https://wordpr
 
 For more details, see [Clearing Caches for Drupal and WordPress](/clear-caches).
 
+## Persistent Cache
+
+Serve your Drupal or WordPress site, even when it's down. If the server is not responding and can't serve a new copy of a page, it will use a cached version instead of displaying an error. With Persistent Cache, the goal is to provide a seamless, uninterrupted experience for the user.
+
+### How Do I Configure This Setting?
+
+On [Drupal](/drupal-cache#drupal-8-performance-configuration) and [WordPress](//wordpress-cache-plugin#pantheon-page-cache-plugin-configuration), you can adjust your CDN edge configuration to serve stale content for a specific amount of time.
+
+
+### Caching Exceptions
```

Comment:
A related suggestion we might want to let users know about (and exactly why the feature is not a "persistent cache") --
The probability that an object will successfully be served stale drops appreciably if its original fresh TTL was less than `3700` seconds. The reason is that such objects are only cached in RAM on Fastly's cache nodes and, as a limited resource, they are more likely to experience cache eviction. In fact, the likelihood that they will experience RAM eviction over the hours-to-days spans needed to retain a large pool of stale resources is extraordinarily high.

So the customer-actionable recommendation here would be to set their TTLs to >= 3700 seconds to fully benefit from this feature.

Fastly reference: https://docs.fastly.com/en/guides/serving-stale-content#why-serving-stale-content-may-not-work-as-expected

Reply:
I added this recommendation to the section